src/EventStore.Core/Services/Storage/StorageScavenger.cs

```diff
@@ -50,9 +50,9 @@ public class StorageScavenger :
 			if (IsAllowed(message.User, message.CorrelationId, message.Envelope)) {
 				lock (_lock) {
 					if (_currentScavenge != null) {
-						message.Envelope.ReplyWith(new ClientMessage.ScavengeDatabaseResponse(message.CorrelationId,
-							ClientMessage.ScavengeDatabaseResponse.ScavengeResult.InProgress,
-							_currentScavenge.ScavengeId));
+						message.Envelope.ReplyWith(new ClientMessage.ScavengeDatabaseInProgressResponse(message.CorrelationId,
+							_currentScavenge.ScavengeId, "Scavenge is currently running"));
```

Comment:
```suggestion
							_currentScavenge.ScavengeId, "Scavenge is already running"));
```

This is a bit subtle, but I think it reads more naturally than 'Scavenge failed: Scavenge is currently running' which on the face of it could seem a bit contradictory